src/Tools/ExternalAccess/FSharp/Internal/Classification/FSharpClassificationService.cs

```diff
@@ -48,5 +47,15 @@ public ClassifiedSpan AdjustStaleClassification(SourceText text, ClassifiedSpan
         {
             return _service.AdjustStaleClassification(text, classifiedSpan);
         }
+
+        public ValueTask<object?> GetDataToCacheAsync(Document document, CancellationToken cancellationToken)
+        {
+            return new();
+        }
```

Comment:
```suggestion
    => new();
```

I'm not sure which way I like, so take how you see fit.